src/node_worker.cc

```diff
@@ -116,9 +116,7 @@ Worker::Worker(Environment* env, Local<Object> wrap, const std::string& url)
     Context::Scope context_scope(context);
 
     // TODO(addaleax): Use CreateEnvironment(), or generally another public API.
-    env_.reset(new Environment(isolate_data_.get(),
-                               context,
-                               nullptr));
+    env_.reset(new Environment(isolate_data_.get(), context));
```

Comment:
```suggestion
    env_ = std::make_unique<Environment>(isolate_data_.get(), context);
```

Reply:
I’d probably prefer to address that along with the main TODO comment here

Reply:
Ok.

Reply:
Ok.